tests/features/interface-default-language.feature

```diff
@@ -4,19 +4,21 @@ Feature: Interface default language.
   As a admin
   I want the interface language to be in the site default language
 
+  @wip
```

Comment:
We lose our entire test coverage for the default admin language. Let's see if we cannot make the new string if it doesn't exist.